### PR TITLE
added tomcat 8.5 support

### PR DESCRIPTION
**Pull request by st0ne-dot-at**
due to deprecation of session manager attribute "maxInactiveInterval"
seel https://tomcat.apache.org/migration-85.html#SessionManager

### CODE REVIEW

**Comment by mesutcelik** [comment]
verify

**Comment by mesutcelik** [comment]
Thanks a lot for the contribution @st0ne-dot-at ! 👏 👏 

I have added some minor comments. Can you please have a look?

@emre-aydin , Can you also review the PR?

Can you also sign Hazelcast Contributor Agreement because that is needed to merge the code into master?
https://hazelcast.atlassian.net/wiki/display/COM/Hazelcast+Contributor+Agreement

File: tomcat85/pom.xml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <name>Hazelcast Tomcat Session Manager(Tomcat8)</name>
```
Comment (emre-aydin) [inline tomcat85/pom.xml]: Should be `Hazelcast Tomcat Session Manager(Tomcat8.5)` to make it clear.

File: tomcat85/pom.xml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    <artifactId>hazelcast-tomcat8-sessionmanager</artifactId>
```
Comment (emre-aydin) [inline tomcat85/pom.xml]: Should be `hazelcast-tomcat85-sessionmanager` as well.

**Comment by st0ne-dot-at** [comment]
@emre-aydin : fixed that

**Comment by emre-aydin** [comment]
verify

**Comment by emre-aydin** [comment]
verify

**Comment by st0ne-dot-at** [comment]
CA signed

**Comment by emre-aydin** [comment]
@st0ne-dot-at I think you missed renaming the following files:
https://github.com/hazelcast/hazelcast-tomcat-sessionmanager/pull/17/files#diff-61ed2d063f74b45e67b0d4698147a770
https://github.com/hazelcast/hazelcast-tomcat-sessionmanager/pull/17/files#diff-c1c2363b3a23cf46f24f76dd0017dba4
https://github.com/hazelcast/hazelcast-tomcat-sessionmanager/pull/17/files#diff-9e107ba0000fa561bde6eae9ee1b80e3
https://github.com/hazelcast/hazelcast-tomcat-sessionmanager/pull/17/files#diff-d8c22d68d94880061616de79ecbb531c

**Comment by emre-aydin** [comment]
verify

**Pull request merged**